### PR TITLE
Completed prep-css-flexbox exercise.

### PR DESCRIPTION
**Pull request by meganbmartinez**
1) Display Flex Screenshot 2021-11-11 223839
2) Flex-Wrap Screenshot 2021-11-11 224039
3) Justify-Content Screenshot 2021-11-12 041255

4) Align-Items Screenshot 2021-11-12 040831

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+
+      .card-deck {
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: Try using an `id` selector here (`#card-deck`).

File: prep-css-flexbox/align-items.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        align-items: center;
+        justify-content: center;
+      }
+
+      .card-container {
+        width: 33.33%;
+        display: flex;
+        min-height: 100vh;
+        align-items: center;
+        justify-content: center;
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: You should be able to remove the `display`, `align-items`, and `justify-content` properties from the `.card-container` and still achieve the desired effect. Although, the `min-height` should be moved to your `#card-deck`.

File: prep-css-flexbox/flex-wrap.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - flex-wrap</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+
+      .card-container {
+        display: inline-flex;
+        flex-wrap: wrap;
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html]: For this part of the assignment, you'll want to put `display` and `flex-wrap` into a rule set that selects the `#card-deck`.

File: prep-css-flexbox/justify-content.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+
+      .card-deck {
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html]: For this rule set, we'll want to use an id selector like `#card-deck`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Megan, this might look like a large review, but some of the suggestions are very similar. Please let me know if I can help out!

**Comment by meganbmartinez** [comment]
1) Align Items Screenshot 2021-11-12 162149
2) Flex Wrap Screenshot 2021-11-12 162149
3) Justify Content Screenshot 2021-11-12 163546

**Review by thebearingedge — APPROVED** [review]
Much better 👍

**Pull request merged**